pkg/executor/hash_table.go

```diff
@@ -152,6 +152,42 @@ func (c *hashRowContainer) GetMatchedRows(probeKey uint64, probeRow chunk.Row, h
 	return matchedRows, err
 }
 
+// GetOneMatchedRow get one matched rows from probeRow.
+func (c *hashRowContainer) GetOneMatchedRow(probeKey uint64, probeRow chunk.Row, hCtx *hashContext) (*chunk.Row, error) {
+	var err error
+	innerEntry := c.hashTable.Get(probeKey)
+	if innerEntry == nil {
+		return nil, err
+	}
+	var matchedRow chunk.Row
+
+	c.chkBuf = nil
```

Comment:
Maybe don't need to reset it to nil for every call of `GetOneMatchedRow`?

Reply:
Indeed.

Reply:
Indeed.